app/views/bets/show.html.erb

```diff
@@ -1,49 +1,73 @@
+<!-- Bet -->
+<hr>
+<div class="row">
+  <div class="col col-lg-6 offset-lg-3">
+    <div class="card text-center">
+      <div class="card-header text-muted">
+        <h4><%= @bet.creator.username %></h4>
+      </div>
+
+      <div class="class-block">
+        <p><h5 class="card-text"><%= @bet.bet %></h5></p>
+      </div>
+
+      <div class="card-footer text-danger">
+        Expires on: <%= @bet.expires_at.strftime('%A, %B %d %Y, %l:%M %P') %>
+      </div>
     </div>
   </div>
-  <hr>
-  <div class="row">
-    <div class="col col-lg-4">
-      <h4>ID</h4>
-    </div>
-    <div class="col col-lg-8">
-      <h4><%= @bet.id %></h4>
-    </div>
-  </div>
-  <div class="row">
-    <div class="col col-lg-4">
-      <h4>Status</h4>
-    </div>
-    <div class="col col-lg-8">
-      <h4><%= @bet.status %></h4>
-    </div>
-  </div>
-  <div class="row">
-    <div class="col col-lg-4">
-      <h4>Creator ID</h4>
-    </div>
-    <div class="col col-lg-8">
-      <h4><%= @bet.creator_id %></h4>
-    </div>
-  </div>
-  <div class="row">
-    <div class="col col-lg-4">
-      <h4>Expires at</h4>
-    </div>
-    <div class="col col-lg-8">
-      <h4><%= @bet.expires_at %></h4>
+</div>
+
+<!-- Users and their wagers -->
+<hr>
+<div class="row">
+  <div class="col col-lg-6 offset-lg-3">
+    <div class="card">
+      <div class="card-header text-muted">
+        <div class="row">
+          <div class="col col-lg-4">User</div>
+          <div class="col col-lg-4">Option #</div>
+          <div class="col col-lg-4">Amount</div>
+        </div>
+      </div>
+      <div class="card-block">
+        <table class="table table-hover">
+          <% @bet.user_bets.each do |ub| %>
+            <tr class="row">
+              <td class="col col-lg-4"><%= ub.user.username %></td>
+              <td class="col col-lg-4"><%= @bet.options.index(ub.bet_option) + 1%></td>
+              <td class="col col-lg-4">$<%= ub.user.waged_on(ub.bet_option) %></td>
+            </tr>
+          <% end %>
+        </table>
+      </div>
     </div>
   </div>
-  <div class="row">
-    <div class="col col-lg-4">
-      <h4>Text</h4>
-    </div>
-    <div class="col col-lg-8">
-      <h4><%= @bet.bet %></h4>
+</div>
+
+<!-- Options -->
+<hr>
+<div class="row">
+  <div class="col col-lg-6 offset-lg-3">
+    <div class="card">
+      <div class="card-header text-muted">
+        <div class="row">
+          <div class="col col-lg-4">Option #</div>
+          <div class="col col-lg-4">Text</div>
+          <div class="col col-lg-4">Option total</div>
+        </div>
+      </div>
+      <div class="card-block">
+        <table class="table table-hover">
+          <% @bet.options.each_with_index do |option, i| %>
+            <tr class="row">
+              <td class="col col-lg-4"><%= i + 1 %></td>
+              <td class="col col-lg-4"><%= option.option_text %></td>
+              <td class="col col-lg-4">$<%= option.option_total %></td>
```

Comment:
:+1: Naming is hard, but I do agree. We dont need to change this now. But definitely something to keep in mind for the future